app/src/main/java/org/mozilla/tv/firefox/architecture/KillswitchLinearLayout.kt

```diff
@@ -71,21 +69,38 @@ class KillswitchLinearLayout : LinearLayout {
             return super.setVisibility(View.GONE)
         }
 
-        val allAllowed = allowedInLocales == KillswitchLocales.All
+        if (allowedInLocales == KillswitchLocales.All) {
+            return super.setVisibility(visibility)
+        }
+
+
         val allowedLocales = (allowedInLocales as? KillswitchLocales.ActiveIn)?.locales
```

Comment:
We have no tests for this null code path: is there a way we can test this so we can guarantee this code doesn't unintentionally break in the future?